trade-finance-manager-api/src/v1/controllers/should-update-deal-from-MIA-to-MIN.js

```diff
@@ -3,12 +3,14 @@ const CONSTANTS = require('../../constants');
 const shouldUpdateDealFromMIAtoMIN = (deal, tfmDeal) => {
   const isMIA = deal.submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA;
 
-  const hasApprovedDecision = (tfmDeal.underwriterManagersDecision
+  if (Object.prototype.hasOwnProperty.call(tfmDeal, 'underwriterManagersDecision')) {
```

Comment:
can we run something like
`if (tfmDeal.underwriterManagersDecision) { ... }`

Reply:
Updated.